src/Components/Popup.js

```diff
@@ -1,7 +1,6 @@
 export default class Popup {
   constructor({ popupSelector }) {
     this._popupElement = document.querySelector(popupSelector);
-    // this.setEventListeners();
   }
 
   open() {
```

Comment:
`    this._popupElement.classList.add("modal__opened");
`

`modal__opened` is an element of block `modal` but according to the task and BEM you need to use a **modificator** with only **one** low dash: `modal_opened`